src/utils/api.js

```diff
@@ -1,100 +1,39 @@
-// import { baseUrl } from "./constants";
-import corgi from "../images/Corgi.png";
-import moose from "../images/Moose.png";
+import { baseUrl, processServerResponse } from "./constants";
 
-export function getItems() {
-  return new Promise((resolve, reject) => {
-    const errorCondition = false;
+export const getArticles = (token) => {
+  return fetch(`${baseUrl}/articles`, {
+    headers: {
+      "Content-Type": "application/json",
+      authorization: `Bearer ${token}`,
+    },
+  }).then((res) => processServerResponse(res));
```

Comment:
**Could be improved**

You can make a special method (function ) for fetching and checking responses **not to duplicate** it in every request:

```js

  function request(url, options) {
    return fetch(url, options).then(processServerResponse)
  }

```

And now you can safely replace `fetch` with `request` and remove repeated checking. And you don’t need to change anything else.